build/exports_test.go

```diff
@@ -0,0 +1,7 @@
+package build
+
+import "github.com/docker/docker/api/types/container"
+
+func GetPhaseHostConfig(phase *Phase) *container.HostConfig {
```

Comment:
I don't love adding a getter to the package's exported interface just for testing. I would rather move the phase tests directly into the `build` package so we can test the unexported field if adding another test endpoint to https://github.com/buildpack/pack/blob/master/build/testdata/fake-lifecycle/phase.go is too annoying (the design of these tests could be simpler).